lib/WP_Auth0_Lock10_Options.php

```diff
@@ -132,6 +132,15 @@ protected function build_settings( $settings ) {
     if (isset($settings['language']) && !empty($settings['language'])) {
       $options_obj['language'] = $settings['language'];
     }
+
+   if ( isset( $settings['wpml_support'] ) && $settings['wpml_support'] == '0' ) {
+      $options_obj['wpml_support'] = null;
```

Comment:
What's the reason for setting `null` explicitly?

Reply:
Nop i do the same that the option gravatar